pages/common/gh-issue.md

```diff
@@ -0,0 +1,28 @@
+# gh-pr
+
+> Manage GitHub issues from the command line.
+> More information: <https://cli.github.com/manual/gh_issue>.
+
+- Print out the issue:
+
+`gh issue view {{issue_number}}`
+
+- Create a new issue in the web browser:
+
+`gh issue create --web`
+
+- List 10 issues with "bug" label:
+
+`gh issue list -L 10 -l "bug"`
```

Comment:
We prefer long argument format:
```suggestion
`gh issue list --limit {{count}} --label {{label}}`
```

Reply:
In here (reffering to what I wrote in a comment) - I wanted to focus on practical example "what the user might want to do", instead of boring listing what are available options - that's what `man pages` and `--help` already do :confused:

Reply:
```suggestion
`gh issue list --limit {{10}} --label "{{bug}}"`
```